libs/solid/skeleton/src/lib/components/base-layout/base-layout.component.html

```diff
@@ -4,12 +4,14 @@
       <div class="right-border">
         <span>Menü</span>
       </div>
-      <img
+      <a href="/">
```

Comment:
Simple solution. Bei mir funktioniert es, aber ich hätte erwartet, dass es über den Router mit `[routerLink]` oder ein `navigateTo` / `new Navigate()` gehen müsste.

Wenn ich durch den Code schaue, ist das jetzt die einzige Stelle, in der ein `<a href...` vorkommt. Bist Du sicher, dass die einfache Hyperlink-Lösung robust ist? Denn so wird die Seite hart neu geladen, aber eigentlich sollte der Router den Content eigentlich dynamisch austauschen, wie es für eine SPA richtig ist.

Eigentlich müsste die gleiche Operation ausgeführt werden wie im Main-Menu "Startseite". Schau doch mal nach, wie das dort gemacht wird.

Reply:
Du hast Recht 😔